src/main/java/teammates/common/util/FieldValidator.java

```diff
@@ -661,67 +645,48 @@ public String getValidityInfoForSizeCappedPossiblyEmptyString(String fieldName,
         return "";
     }
     
-    public String getValidityInfoForTimeFrame(FieldType mainFieldType, FieldType earlierFieldType,
-            FieldType laterFieldType, Date earlierTime, Date laterTime) {
-        
-        Assumption.assertTrue("Non-null value expected", laterFieldType != null);
+    /**
+     * Checks if Session Start Time is before Session End Time
+     * @return Error string if {@code sessionStart} is before {@code sessionEnd}
+     *         Empty string if {@code sessionStart} is after {@code sessionEnd}
+     */
+    public String getInvalidityInfoForTimeForSessionStartAndEnd(Date sessionStart, Date sessionEnd) {
+        return getInvalidtyInfoForFirstTimeIsBeforeSecondTime(sessionStart, sessionEnd,
+                START_TIME_FIELD_NAME, END_TIME_FIELD_NAME);
+    }
+
+    /**
+     * Checks if Session Visibility Start Time is before Session Start Time
+     * @return Error string if {@code visibilityStart} is before {@code sessionStart}
+     *         Empty string if {@code visibilityStart} is after {@code sessionStart}
+     */
+    public String getInvalidityInfoForTimeForVisibilityStartAndSessionStart(Date visibilityStart, Date sessionStart) {
+        return getInvalidtyInfoForFirstTimeIsBeforeSecondTime(visibilityStart, sessionStart,
+                SESSION_VISIBLE_TIME_FIELD_NAME, START_TIME_FIELD_NAME);
```

Comment:
I feed there is a light increase in readability when the closing bracket is on the right side of the opening bracket. Not sure if adding more indentation would violate any checkstyle rules though.